### PR TITLE
11/return file

### LINKED ISSUE #11: Return downloaded file for task use at the server level

### PR DESCRIPTION
**Pull request by pyphilia**
This PR allows for S3 Download Tasks to return the file directly, instead of redirecting the request. Let me know if you find any corner cases where an error should be handled.

close #11

### CODE REVIEW

**Review by louisewang1 — APPROVED** [review]
Thanks for the PR! Everything looks good to me ;-)

File: src/tasks/download-file-task.test.ts [hunk 1]
```diff
@@ -112,17 +127,64 @@ describe('Download File Task', () => {
...
+    //     .mockImplementation(() => ({ promise: jest.fn() }));
+
+    //   const task = new DownloadFileTask(actor, s3Service, input);
+    //   await task.run(handler, log);
+
+    //   // check s3 call
+    //   expect(s3Instance.headObject).toHaveBeenCalledTimes(1);
+    //   expect(s3Instance.getSignedUrlPromise).toHaveBeenCalledTimes(1);
+    //   expect(task.result).toBeTruthy();
+    // });
```
Comment (louisewang1) [inline src/tasks/download-file-task.test.ts]: Maybe use block comment?

**Review by louisewang1 — APPROVED** [review]
Thanks for the PR! All good to me, just a minor comment about the commented code
